wheels/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/pypa/manylinux_2_28_x86_64 
```

Comment:
I'd like to have a musl image in the future. Can you put this file in `wheels-manylinux` so we can have a `wheels-musl` later?